### PR TITLE
Changes to some Exception messages for clarity, consistency and typos - master

### PR DESCRIPTION
**Pull request by jchaves**
After a failing jenkins build, we spent some time trying to understand what was causing the error. This took us to src/org/ods/orchestration/parser/JUnitParser.groovy (L153). We thought that a more specific error message would be useful.

While I was at it, I took a look at all the exceptions thrown, and also changed some others for consistency, and a couple typos.

Hope it helps someone!

### CODE REVIEW

**Review by clemensutschig — APPROVED** [review]

File: src/org/ods/services/OpenShiftService.groovy [hunk 1]
```diff
@@ -804,7 +804,7 @@ class OpenShiftService {
      */
     String patch(String resource, Map<String, ?> patch, String path = null, String project = null) {
         if (!resource) {
-            throw new IllegalArgumentException('You must specify the resource to path')
+            throw new IllegalArgumentException('You must specify the resource to patch') //path or patch??
```
Comment (metmajer) [inline src/org/ods/services/OpenShiftService.groovy:807]: @jchaves patch seems to be correct

**Review by metmajer — CHANGES_REQUESTED** [review]

**Comment by metmajer** [comment]
@jchaves thanks for your contribution!

File: src/org/ods/services/OpenShiftService.groovy (hunk 1 above)
Comment (jchaves) [reply src/org/ods/services/OpenShiftService.groovy]: I thought so. New commit clearing up.

cheers!

**Review by metmajer — DISMISSED** [review]

**Comment by metmajer** [comment]
@jchaves once the workflow has run through, please stash and merge.

**Comment by metmajer** [comment]
@jchaves please amend the Changelog by adding a reference to this PR into the Unreleased category

**Review by metmajer — APPROVED** [review]

**Pull request merged**